src/map/latent_effect.h

```diff
@@ -36,6 +36,7 @@ enum LATENT
     LATENT_HP_OVER_VISIBLE_GEAR     = 46, //hp more than or equal to %, calculated using HP bonuses from visible gear only 
     LATENT_MP_UNDER_PERCENT         = 4,  //mp less than or equal to % - PARAM: MP PERCENT
     LATENT_MP_UNDER                 = 5,  //mp less than # - PARAM: MP #
+    LATENT_MP_OVER                  = 49, //mp greater than # - PARAM: MP #

```

Comment:
@Shadowmh 

If you are using tortosiegit: right click, show log, select your commit where you had it as 55, right click that commit and "reset <myBranchName> to this" in your case branch name will be master. 

A menu will pop up asking which reset type: hard psychically changes all files back. mixed will leave files as is but will change the commit history. At this point you can optionally commit with an amend or just add a new commit for the MAX ID thing. to do this via command line instead, see this link: https://git-scm.com/docs/git-reset

To do this via the crappy github app...don't use that. seriously. Its bad. So is doing edits via the website. :smiley: 

After that you have to force push to over-wright the content on github. In tortoisegit there's a checkbox on the push dialog that says "unknown" which will do the deed. in command line you add a param to your push command which you can read about at the same link above. 

If you need you can jump on IRC and private message me and I can remote in to your PC to show you visually. Sorry I mucked up your work here.